frontend/src/utils/constants.ts

```diff
@@ -40,3 +40,8 @@ export const StorageKey = Object.freeze({
 export const ValidationMessages = Object.freeze({
   REQUIRED: 'This field is required.',
 });
+
+export const ToastTimeout = Object.freeze({
+  ERROR: 5000,
+  SUCCESS: 3000
```

Comment:
INFO and WARN defaults should be defined as well. Consult with Tyler for warning timing, but INFO should be 3000.